testvectors/ecdsa_brainpoolP224r1_sha224_test.json

```diff
@@ -4587,6 +4587,56 @@
           ]
         }
       ]
+    },
+    {
+      "type" : "EcdsaVerify",
+      "source" : {
```

Comment:
Is this the right convention for `source`? Or should I be further subdividing it like `github/davidben/non-minimal-tag`? Not sure what the intention is with how sources are to be managed, and whether it's envisioned to feed into tooling.

Reply:
My vision for the source field is that it should eventually feed into tooling to allow splicing in updates/replacements of a subset of the .json data based on matching the source/version.

If you think you might add more ECDSA test cases in the future for a different class of issue, and would want to update them separately from these, it might make sense to be more specific.

Reply:
Sounds good! Not sure, but doesn't hurt to make it more specific.

Reply:
> (edit: and we should probably capture some advice on this topic in CONTRIBUTING.md too)

I took a crack at porting some of the outcome of this discussion into docs as part of  https://github.com/C2SP/wycheproof/pull/169